Subject: [PATCH] hide release block.
---
Index: config/config.gradle
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/config/config.gradle b/config/config.gradle
--- a/config/config.gradle	(revision ae84abfb655490c8fdff1b875817e6c48dd3045e)
+++ b/config/config.gradle	(date 1720672586941)
@@ -5,16 +5,16 @@
 Properties signProperties = readPropertiesFile("sign.properties")
 
 android {
-    signingConfigs {
-        release {
-            if(signProperties.containsKey("SIGN_KEY_FILE")){
-                storeFile file(signProperties.getProperty("SIGN_KEY_FILE"))
-                storePassword signProperties.getProperty("SIGN_STORE_PASSWORD")
-                keyAlias signProperties.getProperty("SIGN_KEY_ALIAS")
-                keyPassword signProperties.getProperty("SIGN_KEY_PASSWORD")
-            }
-        }
-    }
+//    signingConfigs {
+//        release {
+//            if(signProperties.containsKey("SIGN_KEY_FILE")){
+//                storeFile file(signProperties.getProperty("SIGN_KEY_FILE"))
+//                storePassword signProperties.getProperty("SIGN_STORE_PASSWORD")
+//                keyAlias signProperties.getProperty("SIGN_KEY_ALIAS")
+//                keyPassword signProperties.getProperty("SIGN_KEY_PASSWORD")
+//            }
+//        }
+//    }
     defaultConfig{
         buildProperties.entrySet().forEach {
             if(it.value == "true" || it.value == "false"){
@@ -29,7 +29,7 @@
     }
     buildTypes {
         release {
-            signingConfig signingConfigs.release
+           // signingConfig signingConfigs.release
             manifestPlaceholders = [usesCleartextTraffic:"false"]
         }
         debug {
